### ISSUE TITLE
Fatal error: Unknown table: settings

### ISSUE THREAD

**Issue by randomizeduser** [laurent22/joplin#2793]
Cannot start application. The following error is shown:

![Screenshot at 2020-03-18 10-16-08](https://user-images.githubusercontent.com/61565566/76944567-883ffc80-6901-11ea-96c0-f62bdd89948f.png)

and additionally following errors are shown when starting the app from command line:

> (node:10980) Electron: Loading non context-aware native modules in the renderer process is deprecated and will stop working at some point in the future, please see https://github.com/electron/electron/issues/18397 for more information
> (node:10980) Electron: Loading non context-aware native modules in the renderer process is deprecated and will stop working at some point in the future, please see https://github.com/electron/electron/issues/18397 for more information
> Gtk-Message: 10:15:51.944: GtkDialog mapped without a transient parent. This is discouraged.
> [10968:0318/101749.476154:ERROR:buffer_manager.cc(488)] [.DisplayCompositor]GL ERROR :GL_INVALID_OPERATION : glBufferData: <- error from previous GL command

## Environment

Joplin version: latest as of 2020-03-18
Platform: Linux
OS specifics: Linux Mint 19.3 MATE, kernel 5.5.9-surface 

Kernel from: https://github.com/linux-surface/linux-surface

## Steps to reproduce

1. Install Linux Mint 19.3 MATE + all current updates
2. Install Joplin via 
`wget -O - https://raw.githubusercontent.com/laurent22/joplin/master/Joplin_install_and_update.sh | bash`
3. Start Joplin via `~/.joplin/Joplin.AppImage` or MATE menu.

## Describe what you expected to happen

Application starts.

## Logfile
Following instructions from https://joplinapp.org/debugging/ for "Desktop application" or "CLI application" does not create any more logging. The only message in log.txt is:

> 2020-03-18 10:21:24: "Profile directory: /home/user/.config/joplin-desktop"

**Comment by laurent22** [comment]
Could you share your database.sqlite file, from your profile directory? We can't replicate this bug and having the database might help.

**Comment by randomizeduser** [comment]
Sorry, it was a while ago that I set it up, and did not bother with it again. Now I wanted to fix the issue, and there were actually more steps I took besides the ones in initial description - I just forgot about it.

Back then, after the installation I enabled sync of my existing notes from my nextcloud server incl. encryption key. I guess at this point something went wrong back then during sync. I tried to delete and install `~/.joplin/Joplin.AppImage` over and over again, but that didn't help.

Now I also deleted `~/.config/Joplin` and `~/.config/joplin-desktop`. Before looking at the debug manual I was not aware that these folders exist. I installed the app again now, synced with nextcloud, and it works!!!

Thank you for your quick reply and for this great app you made!
